web/src/components/Layout/Profile/CreatedCollectionList.tsx

```diff
@@ -43,12 +46,27 @@ export default function CreatedCollectionList({
     enabled: !!userId,
   });
 
+  const {
+    posts: selectedCollectionPosts,
+    isFetching: isFetchingPosts,
+    isFetchingNextPage: isFetchingPostsNextPage,
+  } = useInfiniteQueryPosts({
+    key: "user_collection_posts",
```

Comment:
I was thinking about this `key`, and does seem like the userId should be in here.

Reply:
userId is already used inside of useInfiniteQueryPosts for refetching the posts if it changes